### PR TITLE
Workflow trigger for master branch and minor change

### PR DESCRIPTION
**Pull request by Shekharrajak**
- Push and PRs should trigger the work flow

### CODE REVIEW

**Comment by Shekharrajak** [comment]
I am merging this PR to verify it in #74

**Comment by Shekharrajak** [comment]
http://pr-76-numpy.org-newsite.surge.sh/

**Comment by stefanv** [comment]
Nothing on that page seems to load correctly :/

**Comment by stefanv** [comment]
Logs seem OK; although the mime type errors should be addressed.

<details>

```
The script from “https://numpy.org/js/tabs-main.js” was loaded even though its MIME type (“text/html”) is not a valid JavaScript MIME type.
install
The script from “https://numpy.org/js/fresh.js” was loaded even though its MIME type (“text/html”) is not a valid JavaScript MIME type.
install
The script from “https://numpy.org/js/app.js” was loaded even though its MIME type (“text/html”) is not a valid JavaScript MIME type.
install
The script from “https://numpy.org/js/tabs-utils.js” was loaded even though its MIME type (“text/html”) is not a valid JavaScript MIME type.
install
The script from “https://numpy.org/js/jquery.panelslider.min.js” was loaded even though its MIME type (“text/html”) is not a valid JavaScript MIME type.
install
The script from “https://numpy.org/js/fresh.js” was loaded even though its MIME type (“text/html”) is not a valid JavaScript MIME type.
install
Loading failed for the <script> with source “http://numpy.org/js/fresh.js”. install:217:1
The script from “https://numpy.org/js/jquery.panelslider.min.js” was loaded even though its MIME type (“text/html”) is not a valid JavaScript MIME type.
install
Loading failed for the <script> with source “http://numpy.org/js/jquery.panelslider.min.js”. install:218:1
The script from “https://numpy.org/js/tabs-main.js” was loaded even though its MIME type (“text/html”) is not a valid JavaScript MIME type.
install
Loading failed for the <script> with source “http://numpy.org/js/tabs-main.js”. install:223:1
The script from “https://numpy.org/js/tabs-utils.js” was loaded even though its MIME type (“text/html”) is not a valid JavaScript MIME type.
install
Loading failed for the <script> with source “http://numpy.org/js/tabs-utils.js”. install:224:1
loading thebe config 
Object { requestKernel: true, bootstrap: true, binderOptions: {…} }
thebelab.js:89:18
The script from “https://numpy.org/js/app.js” was loaded even though its MIME type (“text/html”) is not a valid JavaScript MIME type.
install
Loading failed for the <script> with source “http://numpy.org/js/app.js”. install:237:1
binder url https://mybinder.org 
Object { ref: "master", binderUrl: "https://mybinder.org", repo: "binder-examples/requirements" }
thebelab.js:343:10
Binder build URL https://mybinder.org/build/gh/binder-examples/requirements/master thebelab.js:383:10
pr-76-numpy.org-newsite.surge.sh content.js:3:13
content.js:4:13
Binder phase: built thebelab.js:405:16
Binder: Found built image, launching...
thebelab.js:417:16
Binder phase: launching thebelab.js:405:16
Binder: Launching server...
thebelab.js:417:16
Binder phase: ready thebelab.js:405:16
Binder: server running at https://notebooks.gesis.org/binder/jupyter/user/binder-examples-requirements-eqjct57s/
thebelab.js:417:16
Starting WebSocket: wss://notebooks.gesis.org/binder/jupyter/user/binder-examples-requirements-eqjct57s/api/kernels/4f734774-d768-4568-a76d-252db346568d default.js:64
Kernel: connected (4f734774-d768-4568-a76d-252db346568d) default.js:1428
```

</details>

**Comment by stefanv** [comment]
It also looks as though the kernel connection happens for every page linked at the top, whereas it should probably only be done for the front page.  Most of the top level items do not link correctly, at least in this deployed version.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -4,10 +4,12 @@ on:
   pull_request:
     branches:
       - newsite
```
Comment (rgommers) [inline .github/workflows/main.yml]: you can remove `newsite` here and further down

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+OR/AND
+
+Describe your changes here
+-->
+
+#### Deployed URL
+
+<!-- replace <pr_number> with your pr number -->
+
+https://pr-<pr_number>-scipy-sphinx-theme-v2.surge.sh
```
Comment (rgommers) [inline .github/pull_request_template.md]: Should be `http://pr-<pr_number>-numpy.org.surge.sh/` instead I think? It actually has `-newsite` in the name as well, that would be good to drop.

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+
+
+#### Brief description of what is fixed or changed
+
+<!-- If this pull request fixes an issue, write "Fixes #NNNN" in that exact
+format, e.g. "Fixes #1234". See
```
Comment (rgommers) [inline .github/pull_request_template.md]: minor: could you change it to `gh-xxxx` here and the line above? (we prefer `gh-` over `#`)

File: .github/workflows/main.yml (hunk 1 above)
Comment (Shekharrajak) [reply .github/workflows/main.yml]: Yeah, I was testing with both branch. I will remove it in your PR.

File: .github/pull_request_template.md (hunk 2 above)
Comment (Shekharrajak) [reply .github/pull_request_template.md]: Oh! my bad I will update it.

File: .github/pull_request_template.md (hunk 3 above)
Comment (Shekharrajak) [reply .github/pull_request_template.md]: Okay! I will do that.

**Comment by Shekharrajak** [comment]
Thanks for the reviews.

**Comment by Shekharrajak** [comment]
> Nothing on that page seems to load correctly :/

I think there is some issue with preloader.  I will look into that meantime or may be @joelachance already know the issue.

**Comment by joelachance** [comment]
It looks like Github Actions runs properly, I'm not finding any errors there. I did notice on the surge deployment we're serving mixed content, which means some things might not load: https://developer.mozilla.org/en-US/docs/Web/Security/Mixed_content/How_to_fix_website_with_mixed_content

I'll continue to look into this as well.

**Comment by joelachance** [comment]
Ok, fixed!

**Pull request merged**